### PR TITLE
Make american-sounding-names Drumpf again

### PR DESCRIPTION
**Pull request by chrisinajar**
Ensure the ability to generate american presidential candidate Donald Drumpf.

### CODE REVIEW

File: data/last.json [hunk 1]
```diff
@@ -99,5 +99,6 @@
     "Butler",
     "Barnes",
     "Fisher",
-    "Bale"
+    "Bale",
+    "Trump"
```
Comment (amaldare93) [inline data/last.json:103]: I think this should actually be spelt "Drumpf", I'd be happy to help you with that.

**Comment by chrisinajar** [comment]
Good point, I updated the PR title to match

EDIT: I have changed my thumb status thusly

File: data/last.json (hunk 1 above)
Comment (chrisinajar) [inline data/last.json:103]: Yeah if you can fix the merge conflict and make that line of code more accurate to Mr Trump's family heritage that would be great.

**Comment by amaldare93** [comment]
fixed in #12 after countless hours

**Comment by chrisinajar** [comment]
Who could possibly count that many hours?

**Pull request closed without merging**